site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/controller/SiteController.java

```diff
@@ -37,22 +38,20 @@ public SiteController(SiteService siteService) {
         return ResponseEntity.status(HttpStatus.CREATED).body(new Response(siteId));
     }
 
-    @PreAuthorize("@authorisationService.authorise('view-site')")
-    @PostAuthorize("@authorisationService.filterUserSites(returnObject.getBody(), #role)")
+    @PostAuthorize("@siteServiceImpl.filterUserSites(returnObject, #role, 'view-site')")
     @GetMapping
-    public ResponseEntity<List<Site>> getSites(@RequestParam(value = "role", required = false) String role) {
-        return ResponseEntity.status(HttpStatus.OK).body(siteService.findSites());
-    }
-
-    @GetMapping("/assigned")
```

Comment:
I like that the new implementation removes the need for this endpoint, which gave site service dependent on practitioner service, was hard to understand and generally yuk.   The new /sites/parents/id endpoint is clearer in my view.